locales/es.js

```diff
@@ -0,0 +1,155 @@
+export const es = {
+  welcome: {
+    title: 'Abra su nueva cuenta bancaria',
+    description_p_1: 'Para abrir una cuenta bancaria, necesitaremos verificar su identidad.',
+    description_p_2: 'Solo tomará un par de minutos.',
+    next_button: 'Verificar identidad',
+  },
+  document_selector: {
+    title: 'Verifique su identidad',
+    hint: 'Seleccione el tipo de documento que desea cargar',
+  },
+  capture: {
+    driving_licence: {
+      front: {
+        title: 'Frente de la licencia',
+        instructions: 'Cargue el frente de la licencia desde su computadora',
+        webcam: 'Coloque el frente de la licencia en el cuadro (será detectado automáticamente)',
+      },
+      back: {
+        title: 'Reverso de la licencia',
+        instructions: 'Cargue el reverso de la licencia desde su computadora',
+        webcam: 'Coloque el reverso de la licencia en el cuadro (será detectado automáticamente)',
+      },
+    },
+    national_identity_card: {
+      front: {
+        title: 'Frente de la tarjeta',
+        instructions: 'Cargue el frente de la tarjeta desde su computadora',
+        webcam: 'Coloque el frente de la tarjeta en el cuadro (será detectado automáticamente)',
+      },
+      back: {
+        title: 'Reverso de la tarjeta',
+        instructions: 'Cargue el reverso de la tarjeta desde su computadora',
+        webcam: 'Coloque el reverso de la tarjeta en el cuadro (será detectado automáticamente)',
+      },
+    },
+    passport: {
+      front: {
+        title: 'Página con la foto del pasaporte',
+        instructions: 'Cargue la página con la foto del pasaporte desde su computadora',
+        webcam: 'Coloque la página con la foto de su pasaporte en el cuadro (será detectado automáticamente)',
+      },
+    },
+    document: {
+      help: '¿Tiene algún problema? Cargue un archivo en su lugar',
+    },
+    face: {
+      title: 'Tòmese una selfie',
```

Comment:
[minor] I think this has the tilde (acento) on the wrong angle (ò instead of ó)

Reply:
Our audience is from South America/Mexico. Apparently in Mexico selfie is feminine.  🤔